storybook/storybook-css/src/components/OrderedList/OrderedList.stories.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { OrderedList } from '@amsterdam/design-system-react/src'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import '@amsterdam/design-system-css/dist/ordered-list/ordered-list.css'
+
+const meta = {
+  title: 'CSS Components/Ordered List',
+  component: OrderedList,
+  args: {
+    children: [
+      <OrderedList.Item key={0}>
+        Dit is het eerste niveau opsommingsteken.
+        <OrderedList>
+          <OrderedList.Item key={1.1}>Dit is het tweede niveau opsommingsteken.</OrderedList.Item>
+        </OrderedList>
+      </OrderedList.Item>,
+      <OrderedList.Item key={1}>Dit is het eerste niveau opsommingsteken.</OrderedList.Item>,
+      <OrderedList.Item key={2}>Dit is het eerste niveau opsommingsteken.</OrderedList.Item>,
+      <OrderedList.Item key={3}>
+        Dit is het eerste niveau opsommingsteken.
+        <OrderedList>
+          <OrderedList.Item key={1.1}>Dit is het tweede niveau opsommingsteken.</OrderedList.Item>
```

Comment:
Zou dit niet 3.1 moeten zijn? Of misschien zelfs 4.1 als je start met 1 ipv 0, misschien iets meer human-friendly.

Reply:
- Voor deze actie hebben uw kinderen een persoonlijke OV-chipkaart nodig. Hebben zij die nog niet, dan kunt u die nu al aanvragen. Ieder kind heeft een eigen OV-chipkaart nodig.
-- [Bretten West](https://nl.wikipedia.org/wiki/Lange_Bretten)
- U kunt hem aanvragen via [ov-chipkaart.nl](https://www.ov-chipkaart.nl/persoonlijke-ov-chipkaart-aanvragen). De kaart kost € 7,50. U krijgt hem na een dag of 5 thuisgestuurd.
- Op de OV-chipkaart kunt u [gratis reizen voor kinderen aanvragen](https://www.amsterdam.nl/verkeer-vervoer/kinderen-gratis-openbaar-vervoer/) vanaf maandag 3 juli 9.00 uur tot uiterlijk 23 november.
- We helpen mensen die er zelf niet uitkomen. Daarvoor zit een speciaal belteam klaar, bereikbaar via 14 020.
-- Zuidwestkwadrant Osdorp Noord
-- Zuidwestkwadrant Osdorp Zuid
- Het product is geldig in alle bussen, trams en metro’s van GVB. Kinderen reizen met ten minste 1 volwassen begeleider.